src/js/main.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
 import { initializeTranslations } from './translations.js';
+import { saveLanguageChoice, saveDifficultyChoice, saveCategoryChoice } from './settings.js'
```

Comment:
Functiile astea nu ar trebui folosite in main.js, nu au legatura cu pagina default. Trebuie lasate doar in settings.js.